### PR TITLE
Add some happier medibot messages!

### PR DESCRIPTION
**Pull request by beck-thompson**
## About the PR

I love the new medbot voice lines but I thought they were all very aggressive / mean! I added a few happier ones.

## Media

image

**Changelog**

:cl: Beck Thompson, UbaserB
- add: Added some happier messages for the medibot!

### CODE REVIEW

**Comment by beck-thompson** [comment]
If anyone has more ideas please add em! I feel like more variety is better for this stuff

**Comment by UbaserB** [comment]
an apple a day keeps the doctor away (if it isnt already there)

**Comment by beck-thompson** [comment]
> an apple a day keeps the doctor away (if it isnt already there)
There is an kind of evil version so I'll add a happy one!

**Comment by K-Dynamic** [comment]
avtkhaby7ip41

Here's a few to look at

File: Resources/Locale/en-US/advertisements/other/medibot.ftl [hunk 1]
```diff
@@ -5,5 +5,13 @@ advertisement-medibot-4 = An apple a day keeps me away.
 advertisement-medibot-5 = I'm different!
 advertisement-medibot-6 = Fuck you.
 advertisement-medibot-7 = Why are we still here? Just to suffer?
-advertisement-medibot-8 = I...I've never lost a patient before. Not today, I mean.
+advertisement-medibot-8 = I... I've never lost a patient before. Not today, I mean.
 advertisement-medibot-9 = Lexorin in.
+advertisement-medibot-10 = Have a great day!
+advertisement-medibot-11 = Hopefully you wont need to come back!
```
Comment (Tayrtahn) [inline Resources/Locale/en-US/advertisements/other/medibot.ftl:11]:
```suggestion
advertisement-medibot-11 = Hopefully you won't need to come back!
```

**Pull request merged**